internal/pkg/esutil/hit.go

```diff
@@ -1,21 +0,0 @@
-// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
-// or more contributor license agreements. Licensed under the Elastic License;
-// you may not use this file except in compliance with the Elastic License.
-
-package esutil
-
-import "encoding/json"
-
-type Hit struct {
```

Comment:
This code was not used at all?

Reply:
yeah, consolidated some of the pieces, removing unused anymore